src/components/modules/InfoBox/index.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { PureComponent } from 'react';
+import { StyleSheet, View, Text, ImageBackground, Dimensions } from 'react-native';
+import { ModuleBox } from '@/components/modules';
+import { Image, Container, HtmlText } from '@/components/primitives';
+import { Responsive } from '@/components/utils';
+import { COLOR_MAP } from '@/constants';
+import { BREAKPOINTS } from '@/constants/index';
+import { InfoBoxData, OFFSET_DIRECTION } from '@/types';
+
+interface Props {
+    data: InfoBoxData[];
+}
+
+type State = {
+    readText: string;
+    width: number;
+    height: number;
+    windowWidth: number;
+    imgHeight: number;
+}
+
+export default class InfoBox extends PureComponent<Props, State> {
+    state = {
+        readText: 'Read More',
+        width: 0,
+        height: 0,
+        windowWidth: 0,
+        imgHeight: 0,
+    }
+
+    private mobileInfoBox = React.createRef;
```

Comment:
I think this should be React.createRef(), but it doesn't look like it's actually used. Can it be removed?

Reply:
Removed.